frontend/src/containers/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import { houseIssuesFetchData} from './actions';
+
+import {HouseList} from '../../components/House';
+
+class Dashboard extends React.Component {
+  componentDidMount = () => {
+    this.props.fetchData( '/api/v1/house-issues');
+  };
+  render = () => {
```

Comment:
You talk about me being badly spaced i.e. too much. But too little can also be annoying to read. As a rule I would put spaces between functions and separate blocks of code atleast. Maybe not 5 like me. But maybe one ?

Reply:
I think we should add style checks into linter. Added a line.